web/app/scripts/state/geographystate-service.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Geography state control - changes to the private vars which define this state
+ *  are broadcast to rootscope for use by controllers
+ */
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function GeographyState($window, $log, $rootScope, Geography) {
+        var defaultParams, selected, options;
+        var _ = $window._;
```

Comment:
Is this for `jshint` purposes? I think that can be solved by setting it in the jshint config file.